src/chocolatey/infrastructure.app/services/ChocolateyPackageInformationService.cs

```diff
@@ -47,11 +48,16 @@ public ChocolateyPackageInformationService(IFileSystem fileSystem, IRegistryServ
         }
 
         public ChocolateyPackageInformation get_package_information(IPackage package)
+        {
+            return get_package_information(package, new ChocolateyConfiguration());

```

Comment:
This is probably fine, but I would be a bit more complete by adding the config into the service itself.

Reply:
That way you don't need to pass config around to the functions.